### PR TITLE
Make package compatible with probot v10

### LINKED ISSUE #43: Upgrade to Probot v10
Hello Lambda friends,

we just released Probot v10 and I'd like to upgrade `@probot/serverless-lambda` for compatibility. Would any existing user/contributor would like to collaborate on that with me?

More importantly, I'd like to make this package obsolete with Probot v11. I'm not sure how it would look like, but making Probot play nicely with serverless environments is our priority for the next release. Please join the discussion at
https://github.com/probot/probot/issues/1286

### PR DESCRIPTION
**Pull request by scolandrea**
PR adds compatibility with probot v10.

Credits on @soberstadt

resolves #43

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,15 +1,15 @@
-const { createProbot } = require('probot')
-const { resolve } = require('probot/lib/resolver')
-const { findPrivateKey } = require('probot/lib/private-key')
+const { Probot } = require('probot')
+const { resolve } = require('probot/lib/helpers/resolve-app-function')
+const { findPrivateKey } = require('probot/lib/helpers/get-private-key')
```
Comment (gr2m) [inline index.js:3]: I'm not happy that we use internal APIs, I'll see if we can make these methods official exports at the root level so things won't break if Probot's internal file structure changes.

**Review by gr2m — COMMENTED** [review]
Looks good 👍🏼 Did you have a chance to test if it still works /cc @soberstadt

**Comment by scolandrea** [comment]
I'm trying to test it on AWS but I'm dealing with a problem with my account. Hopefully I will be able to test it over the weekend :)

**Comment by soberstadt** [comment]
I tested this on my probot and it worked 🎉
image

**Comment by scolandrea** [comment]
Awesome!! Glad to hear that!

**Comment by gr2m** [comment]
Sweet, thank you for checking!

**Pull request merged**